.changeset/three-buses-vanish.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": minor
```

Comment:
I hope I did this right by setting this a minor version bump? Let me know if this change should be a major version bump. I thought minor version bump because this PR isn't introducing any breaking changes.

Reply:
A minor bump is perfect for this! 👍